server/plugins/logger.js

```diff
@@ -3,88 +3,33 @@ module.exports.register.attributes = {
   name: 'hoodie-local-logger'
 }
 
-var qs = require('querystring')
-
-var log = require('npmlog')
-var Squeeze = require('good-squeeze').Squeeze
-
-function GoodEvent (events, config) {
-  this._callback = config.callback || function () {}
-  this._filter = new Squeeze(events)
-}
-
-GoodEvent.prototype.init = function (readstream, emitter, callback) {
-  readstream.pipe(this._filter).on('data', this._callback)
-  callback()
-}
-
 function register (server, options, next) {
   server.register({
     register: require('good'),
     options: {
-      opsInterval: 30000,
-      reporters: [{
-        reporter: GoodEvent,
-        config: {
-          callback: logger
-        },
-        events: {
-          ops: '*',
-          response: '*',
-          log: '*',
-          error: '*',
-          request: '*',
-          wreck: '*'
-        }
-      }]
+      ops: {
+        interval: 30000
+      },
+      reporters: {
+        hoodieReporter: [
+          {
+            module: 'good-squeeze',
+            name: 'Squeeze',
+            args: [
+              {
+                ops: '*',
+                response: '*',
+                log: '*',
+                error: '*',
+                request: '*'
+              }
+            ]
+          },
+          {
+            module: '../../../server/plugins/good-hoodie'
```

Comment:
I agree with @capellini that using `path` will be a lot more cleaner. I had assumed that we might not change the location of the file and hence, didn't want to require additional modules.

Btw, thanks for writing the tests @capellini :+1: